go-selinux/selinux_linux.go

```diff
@@ -1102,7 +1102,18 @@ func chcon(fpath string, label string, recurse bool) error {
 	}
 
 	if !recurse {
-		return setFileLabel(fpath, label)
+		err := lSetFileLabel(fpath, label)
+		if err == nil {
+			return nil
+		}
+		if errors.Is(err, os.ErrNotExist) {
+			return err
+		}
+		flabel, _ := lFileLabel(fpath)
+		if flabel == label {
```

Comment:
I _guess_ it's ok(isn), but we don't check the error here, so technically can't assume `flabel` is an ok value